.trunk/trunk.yaml

```diff
@@ -14,8 +14,8 @@ runtimes:
   enabled:
     - node@18.12.1
     - python@3.10.8
-    - rust@1.81.0
     - ruby@3.1.0
+    - rust@2024-05-01
```

Comment:
Rust couldn't be upgraded to a more recent nightly version since it requires a version of `GLIBC` that's newer than the version of Ubuntu our CI runners use.